### DISCUSSION TITLE
include pre-made virtual machines with releases

### DISCUSSION THREAD

**Discussion by pablo5425** [Berny23/LD-ToyPad-Emulator#145] [Ideas]
i think including pre-made virtual machines as vdi or img files could help with making the whole setup basically plug & play
idk if there is a size limit for files that can be uploaded into releases tho

**Comment by Berny23** [comment]
Maybe with Docker images, we are still figuring out if it is possible.

**Comment by VladimirKuletski** [comment]
I'm trying to finish Ansible playbook to speed up manual validations on Pi Zero W to keep up with proposed changes. If this works well can create hosted GitHub runner later for automated validation.

Will try to build a container later. But I would expect Node vs Python version inconsistency there also.

And I haven't touched VDIs at all. But it can be also automated. Will see if I can make it soon.
